### ISSUE TITLE
Contributing to server

### ISSUE THREAD

**Issue by thetsajeet** [Rahilsiddique/codeAck#14]
Are contributions to server accepted? In any case to run the server, need to setup the env variables for it. Could you provide those as an example.env file to check the format of the env variables to setup.

**Comment by adityayadav71** [comment]
Yes contributions to server are accepted. I have attached an example_env text file. If there are any questions, feel free to ask 🙂.
[example_env.txt](https://github.com/Rahilsiddique/codeAck/files/9739458/example_env.txt)
